src/models/meer5/README.md

```diff
@@ -0,0 +1,37 @@
+# Meerkat (meer5)
+
+- [External Overview](./external-overview.md)
+- [Internal Overview](./internal-overview.md)
+- [Parts & Repairs](./repairs.md)
+
+![Short Meerkat](./img/meer5-short.png)
+
+The System76 Thelio Massive is a desktop with the following specifications:
```

Comment:
Thelio Massive? Are these specs correct?

Reply:
Hey, meerkats can dream, can't they? 

Thanks for catching that. I was copying chunks of Massive's docs to use as a template and forgot to update that. The specs are correct.